src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -915,6 +1258,41 @@ async def _handle_update_config(self, request_id: str, data: RTVIUpdateConfig):
         await self._update_config(RTVIConfig(config=data.config), data.interrupt)
         await self._handle_get_config(request_id)
 
+    async def _handle_update_context(self, data: RTVIAppendToContextData):
+        if data.run_immediately:
+            await self.interrupt_bot()
+        frame = LLMMessagesAppendFrame(messages=[{"role": data.role, "content": data.content}])
+        await self.push_frame(frame)
+        if data.run_immediately and self._context_aggregator:
+            # If specified, immediately push the full context frame to trigger an LLM run.
+            match data.role:
+                case "user":
+                    frame = self._context_aggregator.user().get_context_frame()
+                    await self.push_frame(frame)
+                case "assistant":
+                    frame = self._context_aggregator.assistant().get_context_frame()
+                    await self.push_frame(frame)
+                case _:
+                    logger.warning(f"Unknown role {data.role} in RTVIAppendToContext")
```

Comment:
I don't think we need to distinguish between the role here, I believe both should return the same frame.